helpdesk_mgmt_merge/wizard/helpdesk_ticket_merge.py

```diff
@@ -69,9 +71,21 @@ def merge_tickets(self):
         merged_tickets = self.ticket_ids - self.dst_ticket_id
         self._merge_followers(merged_tickets)
         for ticket in merged_tickets:
-            self._add_message("to", self.dst_ticket_id.number, ticket)
-        ticket_numbers = ", ".join(merged_tickets.mapped("number"))
+            self._add_message(
+                "to",
+                f"""<a href="/web#id={self.dst_ticket_id.id}&model=helpdesk.ticket">
+                {self.dst_ticket_id.number}</a>""",  # noqa: B950
```

Comment:
If not, you are introducing the spaces and the line break in the string:

```suggestion
                f"<a href="/web#id={self.dst_ticket_id.id}&model=helpdesk.ticket">"
                f"{self.dst_ticket_id.number}</a>",  # noqa: B950
```

Anyway, there's a method for getting this without effort:

```suggestion
                self.dst_ticket_id._get_html_link(self.dst_ticket_id.number),
```

and the same for the rest.

Reply:
Oh, I had no idea the method existed. I used it because it’s cleaner. Thanks!